src/main/java/com/mycompany/app/App.java

```diff
@@ -10,6 +10,8 @@ public static void main(String[] args) {
         System.out.println(s);
 
         Object b = null;
-        b.notify();
+        if (b != null) {
```

Comment:
Remove parentheses

Reply:
Fixed in b57f62e9a59128368fb36135381857b7c12bec98